### PR TITLE
MRG: Some tiny improvements to montage-related docstrings

### PR DESCRIPTION
**Pull request by hoechenberger**

### CODE REVIEW

File: mne/io/meas_info.py [hunk 1]
```diff
@@ -214,11 +214,16 @@ def set_montage(self, montage, match_case=True, match_alias=False,
...
+
+        See Also
+        --------
+        make_standard_montage
+        make_dig_montage
+        read_custom_montage
 
         Notes
         -----
-        Operates in place.
```
Comment (hoechenberger) [inline mne/io/meas_info.py:227]:
```suggestion
```

File: mne/utils/docs.py [hunk 2]
```diff
@@ -1860,8 +1860,6 @@ def _reflow_param_docstring(docstring, has_first_line=True, width=75):
     channel positions from the montage. Valid strings are the names of the
     built-in montages that ship with MNE-Python; you can list those via
     :func:`mne.channels.get_builtin_montages`.
-    You may also want to consult the documentation of
-    :class:`mne.channels.DigMontage` for additional information.
```
Comment (hoechenberger) [inline mne/utils/docs.py:1864]: I don't think this is helpful at all, as basically one of the first things the DigMontage docs says is, "don't do this at home, ask an expert function to do it for you!" essentially sending you running around in circles. So I dropped it.

File: mne/channels/montage.py [hunk 3]
```diff
@@ -1646,7 +1646,6 @@ def make_standard_montage(kind, head_size='auto'):
 
     See Also
     --------
-    DigMontage
     make_dig_montage
```
Comment (hoechenberger) [inline mne/channels/montage.py:1649]:
```suggestion
    get_builtin_montages
    make_dig_montage
```

**Review by agramfort — APPROVED** [review]

**Comment by agramfort** [comment]
thx @hoechenberger

**Pull request merged**